### ISSUE TITLE
Still doesn't work out of `/opt` due to colors.styles file.

### ISSUE THREAD

**Issue by emctague** [kotajacob/wal_steam#29]
Hey, it still doesn't work from the AUR package, because it's trying to update a file called `colors.styles` in the resources directory:

```
➜  wal_steam -w
Metro Wal found
Reading colors
Patching new colors
No file to remove
Traceback (most recent call last):
  File "/opt/wal_steam/wal_steam.py", line 236, in <module>
    main(arguments)
  File "/opt/wal_steam/wal_steam.py", line 232, in main
    makeStyle(colors)
  File "/opt/wal_steam/wal_steam.py", line 66, in makeStyle
    f_name = open(newColors, 'w')
PermissionError: [Errno 13] Permission denied: '/opt/wal_steam/resources/colors.styles'
```

Perhaps we'll just have to wait until you implement the permanent fix before the AUR package will work.

**Comment by kotajacob** [comment]
Ahh yea I see, I'll work on that now. Shouldn't take all too long.

**Comment by emctague** [comment]
Thank you! Sorry for causing so much trouble by making an AUR package :P

**Comment by kotajacob** [comment]
No honestly thanks for making it. This needed to happen anyway. This pull request should fix the issue.
https://github.com/kotajacob/wal_steam/pull/30

**Comment by kotajacob** [comment]
I merged #30 after doing some more testing and released version 1.1.0

It will store data in ~/.cache/wal_steam now and detect if the skin or the cache is missing and re-download, patch and copy over the skin!
